.github/workflows/deployment.yml

```diff
@@ -12,17 +12,37 @@ jobs:
     outputs:
       version: ${{ steps.set_suffix.outputs.version }}
       suffix: ${{ steps.set_suffix.outputs.suffix }}
+      version_changed: ${{ steps.changes.outputs.version_changed }}
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
           python-version: "3.x"
 
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check if VERSION file is modified compared to main
+        uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          base: main
+          filters: |
+            version_changed:
+              - 'src/mrpro/VERSION'
+
+      - name: Check if files in docker or the toml file has been modified
```

Comment:
I don't think this part is needed

Reply:
Copy/Pasted it in for debugging and forgot to remove again. 🙄